packages/host/app/services/sessions-service.ts

```diff
@@ -78,4 +87,85 @@ export default class SessionsService extends Service {
       }
     }
   }
+
+  async getRealmToken(
+    realmURL: URL,
+    skipCache?: boolean,
```

Comment:
Can this method figure out whether to skip cache by itself (and not tell it to)? It could check if the jwt is about to expire or did expire and skip the cache in that case

Reply:
There is also a case where the JWT is still in valid period but the realm owner update the user permissions. So, this `skipCache` parameter is used to handle that case.